test/gov/gov-combined.sh

```diff
@@ -228,6 +228,10 @@ then
   fi
 fi
 
+if [ "${EXCH_APP_HOST}" = "http://exchange-api:8080/v1" ]; then
+	./swagger_validation.sh
```

Comment:
I think you need a $? check here to stop the test if there is a failure. Otherwise, there are so many messages that come out of e2edev that no one will ever notice the error.

Reply:
changed the script to external script non integrating with make test or gov combine